packages/core/strapi/lib/services/entity-service/types/index.d.ts

```diff
@@ -18,24 +40,118 @@ export interface EntityService {
       | 'publicationState'
       | 'plugin'
     >
-  ): Promise<Attribute.GetValues<TContentTypeUID>[]>;
+  ): Promise<Result<TContentTypeUID>[]>;
+
   findOne<TContentTypeUID extends Common.UID.ContentType>(
     uid: TContentTypeUID,
     entityId: number,
-    params?: EntityService.Params.Pick<TContentTypeUID, 'fields' | 'populate'>
-  ): Promise<Attribute.GetValues<TContentTypeUID> | null>;
+    params?: Params.Pick<TContentTypeUID, 'fields' | 'populate'>
+  ): Promise<Result<TContentTypeUID> | null>;
+
   delete<TContentTypeUID extends Common.UID.ContentType>(
     uid: TContentTypeUID,
     entityId: number,
-    params?: EntityService.Params.Pick<TContentTypeUID, 'fields' | 'populate'>
-  ): Promise<Attribute.GetValues<TContentTypeUID> | null>;
+    params?: Params.Pick<TContentTypeUID, 'fields' | 'populate'>
+  ): Promise<Result<TContentTypeUID> | null>;
+
   create<TContentTypeUID extends Common.UID.ContentType>(
     uid: TContentTypeUID,
-    params?: EntityService.Params.Pick<TContentTypeUID, 'data' | 'files' | 'fields' | 'populate'>
-  ): Promise<Attribute.GetValues<TContentTypeUID>>;
+    params?: Params.Pick<TContentTypeUID, 'data' | 'files' | 'fields' | 'populate'>
+  ): Promise<Result<TContentTypeUID>>;
+
   update<TContentTypeUID extends Common.UID.ContentType>(
     uid: TContentTypeUID,
     entityId: number,
-    params?: EntityService.Params.Pick<TContentTypeUID, 'data' | 'files' | 'fields' | 'populate'>
-  ): Promise<Attribute.GetValues<TContentTypeUID>>;
+    params?: Params.Pick<TContentTypeUID, 'data' | 'files' | 'fields' | 'populate'>
+  ): Promise<Result<TContentTypeUID>>;
+
+  findPage<TContentTypeUID extends Common.UID.ContentType>(
+    uid: TContentTypeUID,
+    params?: Params.Pick<
+      TContentTypeUID,
+      'fields' | 'filters' | 'pagination' | 'sort' | 'populate' | 'publicationState' | 'plugin'
+    >
+  ): Promise<PaginatedResult<TContentTypeUID>>;
+
+  clone<TContentTypeUID extends Common.UID.ContentType>(
+    uid: TContentTypeUID,
+    cloneId: number,
+    params?: Params.Pick<TContentTypeUID, 'fields' | 'populate' | 'data' | 'files'>
+  ): Promise<Result<TContentTypeUID> | null>;
+
+  deleteMany<TContentTypeUID extends Common.UID.ContentType>(
+    uid: TContentTypeUID,
+    params: Params.Pick<TContentTypeUID, 'populate'>
```

Comment:
```suggestion
    params: Params.Pick<TContentTypeUID, 'filters' | 'populate'>
```
Not sure how we missed that, without filters we'd just be deleting all :)